wonderland/src/index.js

```diff
@@ -95,14 +97,23 @@ export class ZestyBanner extends Component {
         .then(value => {
           this.zestyNetworking = Object.assign({}, value);
           this.startLoading();
+          if (isBeta) {
+            setInterval(this.startLoading.bind(this), 30000);
```

Comment:
Can the `30000` be a descriptive constant here

Reply:
Done